scripts/scr_marine_struct/scr_marine_struct.gml

```diff
@@ -2430,6 +2397,19 @@ function TTRPG_stats(faction, comp, mar, class = "marine", other_spawn_data={})
 			};
 		return equip_data;
 	}
+	static equipment_maintenance_burden = function(){
+		var burden = 0.0;
+		burden+=get_armour_data("maintenance");
+		burden+=get_gear_data("maintenance");
+		burden+=get_mobility_data("maintenance");
+		burden+=get_weapon_one_data("maintenance");
+		burden+=get_weapon_two_data("maintenance");
+		if (has_trait("tinkerer")){
+			burden *= 0.33;
```

Comment:
I have an idea:
Can we make it that technology stat of the marine that holds a weapon, reduces the maintenance burden?
Like, the techmarines (and techpriests) would be best at maintaining stuff obviously, but this should give the player another consideration on to what marine give the powerful weapons of the chapter, as - higher technology score, the less maintenance the precious weapon needs.
So the player gains another layer to min-max - he can give the best tools for best warriors for battle, but during calmer times, give them to the most tech-inclined marines to lower maintenance costs.

Doing that will ensure that technology stat is relevant for all marines to an extent, and not only techmarines.